src/pages/login/index.js

```diff
@@ -45,6 +57,21 @@ const Login = () => {
   }, [isSuccess, user, push]);
 
   useEffect(() => resetErrors, [resetErrors]);
+
+  const switchTab = dataFromChild => {
+    if (dataFromChild === 'CONTACT') {
```

Comment:
Here for example 😄 we could use the constant defined in the other file if we extract it to somewhere else